### DISCUSSION TITLE
64 bit Image

### DISCUSSION THREAD

**Discussion by R101G** [rern/rAudio#274] [Testing]
@rern,

Is there a future for the 64 bit image, or should I switch to the latest release?

**Comment by klassikmann** [comment]
I've been using aarch64 since it was first offered with almost daily updates (now: 5.10.25-4-ARCH aarch64). No problems to date that could be traced back to aarch64. But I cannot estimate where problems could still be: I don't use WiFi or any of the optional features.
My system: RPi 3B + HifiBerry Digi+ Pro

**Reply by R101G** [reply]
I too have been using aarch64 without problems, but the option to download that image seems to have gone now, which is what prompted my question. 

I do not intend to change anything unless the updates stop.

**Reply by klassikmann** [reply]
_> but the option to download that image seems to have gone now_
On https://github.com/rern/rAudio-1/releases you still find the pre-release:
"rern released this on 23 Feb" but you see "Kernel 5.10.25-4 aarch64" which is brand-new.
OTOH I use https://github.com/rern/rOS to generate an aarch64 installation.

**Reply by R101G** [reply]
Thanks - I didn't look that far down the list!
